src/bandersnatch/simple.py

```diff
@@ -32,17 +48,34 @@ class InvalidSimpleFormat(KeyError):
     pass
 
 
+class InvalidDigestFormat(ValueError):
+    """We don't have a valid digest choice from configuration"""
+
+    pass
+
+
 def get_format_value(format: str) -> SimpleFormat:
     try:
         return SimpleFormat[format.upper()]
     except KeyError:
-        valid_formats = [v.name for v in SimpleFormat].sort()
+        valid_formats = sorted([v.name for v in SimpleFormat])
```

Comment:
Just out of curiosity, why did this change need to happen? Did you need a new list for some reason, I believe this is slower (not that it really matters here)

Reply:
I didn't consider speed to be honest, but that is good to know. `valid_formats` would never get the intended value here because `list.sort()` returns `None` after modifying the object in-place.

Reply:
Yup, you are right here. I missed it. Thanks for the fix.